### ISSUE TITLE
Pharo 9 64-bit image is unstable on Windows 10 Enterprise

### ISSUE THREAD

**Issue by PowellDean** [guillep/pharo-core#9737]
Originally posted under Issue #9730

I have installed the latest PharoLauncher product on two separate Windows 10 machines:

    Machine 1: Windows 10 Enterprise Edition - Version 1909 (OS Build 18363.1679)
    Machine 2: Windows 10 Home Edition - Version 2004 (OS Build 19041.1052)

On BOTH machines I installed the current version of the PharoLauncher, as downloaded from the pharo.org website. I then installed the XML parsing libraries (XMLDOMParser and friends). I then installed the package I've been working on via Monticello.

On Machine 2: No issues at all running either the Pharo 9 32-bit or Pharo 9 64-bit image. External packages both loaded without issue, and my package was able to connect to an HTTPS: endpoint without any problem. Again, this is the case for BOTH 32- and 64-bit Pharo 9 images. The Windows 10 Home Edition experienced no issues.

On Machine 1:
Upon first launching the 64-bit image (via PharoLauncher), the main window appeared, but the Pharo image was completely unresponsive for ~30 seconds. After that time, the following error message popped up:

Information
cannot update #appearance#current
Exception: ConnectionTimeout: Cannot connect to 140.82.113.3:443

After this, the image was responsive again and I was able to load both the XML parsing libraries (XMLDOMParser and friends) and my package through Monticello.

Upon running my package code, the system again becaome completely unresponsive for ~25-30 seconds, after which a walkback window appeared outlining the following:

    Class Socket
    Method: connectTo: hostAddress port: port waitForConnectionFor: timeout
    Error message: signal: ConnectionTimedOut. Cannot connect to xx.xx.xx.xx:443

On a whim, I launched the PharoLauncher "as Administrator", but I encountered the same results.
I should point out that the Windows 10 Enterprise Edition is my company laptop and is subject to company-installed Group Policy settings. However, I can confirm that the Pharo 8.0 Old Stable images (both 32- and 64-bit) run perfectly on this laptop. The code I wrote connects to the remote HTTPS API endpoint. I have full Local Admin privileges on this laptop.

I should point out that the same issue as opened in #9600 are also present in the 64-bit image as downloaded from the Launcher.

Thank you.

Dean Powell
Edmonton, Canada

**Comment by JanBliznicenko** [comment]
Hello, it seems Pharo is not able to connect to GitHub.
You may try connecting somewhere else, like to Google:
```
socket := Socket newTCP.
ip := NetNameResolver addressForName: 'google.com' timeout: 5.
socket connectTo: ip port: 80 waitForConnectionFor: 5.
self assert: socket isConnected.
self inform: 'Connected successfully'.
socket closeAndDestroy.
```

Also, this might (or not) be related to #9990. On the affected image, try getting result (run via Print it or Inspect it) of following code to tell you how many sockets are open in Pharo at the time these connections could not be made: 
```
Socket allInstances count: #isConnected.
```

Jan

**Comment by pavel-krivanek** [comment]
Please, can you test the latest release if the issue is still valid on your machine?

**Comment by PowellDean** [comment]
Hi:

I changed laptops recently but have an identical Windows version running on
it. After downloading and installing the newest Pharo version I did a few
different tests with both the regular laptop keyboard (Lenovo Thinkbook)
and a connected external keyboard. I couldn't get the issue to re-appear
with either of the keyboards I tried. The external one has extra keys, for
example "Scroll Lock" and "Print Scrn", but the status of "Scroll Lock" had
no effect. In all cases the backspace and delete keys worked just fine.

The old laptop was an HP Elitebook, with a slightly different keyboard
layout. I believe this laptop toggled certain Windows settings with odd key
combinations that weren't readily obvious when typing. It's possible I
mis-hit a key while also depressing CTRL or ALT or some combination that
set something.
